api/core/throttling.py

```diff
@@ -0,0 +1,16 @@
+from core.constants import SDK_ENDPOINTS
+from django.conf import settings
+from django.core.cache import caches
+from rest_framework.request import Request
+from rest_framework.throttling import UserRateThrottle
+
+
+class AdminRateThrottle(UserRateThrottle):
+    cache = caches[settings.ADMIN_THROTTLE_CACHE_NAME]
+    scope = "admin"
+
+    def allow_request(self, request: Request, view: object) -> bool:
+        path = request.path.rstrip("/")
+        if path in SDK_ENDPOINTS:
+            return True
```

Comment:
I guess the only way to resolve the issue above regarding the `/api/v1/identities/{identifier}` endpoint would be to use `startswith` or regex here which would complicate things slightly I guess. What do you think?